src/vm/contexts.rs

```diff
@@ -1107,11 +1122,22 @@ impl<'a, 'b> Environment<'a, 'b> {
             let memory_use = contract_string.len() as u64;
             self.add_memory(memory_use)?;
 
+            let version = ClarityVersion::default_for_epoch(
```

Comment:
This code will eventually only be if the contract doesn't supply a `(clarity-version VERSION)` pragma, right?

Reply:
Yep, that's correct